ansible/group_vars/all.yml

```diff
@@ -52,6 +52,8 @@ dev_users:
 
 django_port: 9010
 riakcs_port: 8080
+temp_riakcs_proxy_port: 8081
+riakcs_proxy_port: "{{ temp_riakcs_proxy_port if 'BLOB_DB_MIGRATING_FROM_S3_TO_S3' in localsettings else riakcs_port }}"
```

Comment:
I think it would be clearer if this logic was in `deploy_proxy.yml` rather than some here and some there.